patches/0005-Add-CNG-crypto-backend.patch

```diff
@@ -1087,26 +1091,26 @@ index a0548a7f9179c5..ae6117a1554b7f 100644
  package x509
  
 diff --git a/src/go.mod b/src/go.mod
-index a0f4b8bb8cae1d..5745a76af48510 100644
+index eea762d89ae327..0ebd229e72b1ef 100644
 --- a/src/go.mod
 +++ b/src/go.mod
 @@ -4,6 +4,7 @@ go 1.22
  
  require (
- 	github.com/golang-fips/openssl/v2 v2.0.0-rc.3.0.20231108174110-41b6eb24da28
-+	github.com/microsoft/go-crypto-winnative v0.0.0-20231013074141-ebaf9de20b54
+ 	github.com/golang-fips/openssl/v2 v2.0.0-rc.3.0.20240109234540-66bdd798d334
++	github.com/microsoft/go-crypto-winnative v0.0.0-20240109184443-a968e40d3103
```

Comment:
This is a commit currently in the https://github.com/microsoft/go-crypto-winnative/pull/52 dev branch. If I make sure to use a merge commit rather than squash for that PR, this should be ok to merge.